test/controllers/surveys_controller_test.rb

```diff
@@ -14,11 +15,15 @@ class SurveysControllerTest < ActionDispatch::IntegrationTest
   end
 
   test "admin can see all surveys" do
-    assert false
+    sign_in @admin
+    get surveys_url
+    assert_equal @survey.length, Survey.all.count
```

Comment:
Maybe we can bullet proof this a bit more, we're only testing with one survey at the moment...

Reply:
Many thanks @spencerldixon I will look into this.